programs/governance-rewards/src/instructions/reclaim/reclaim_user_data.rs

```diff
@@ -0,0 +1,34 @@
+use anchor_lang::prelude::*;
+
+use crate::{
+    error::GovernanceRewardsError,
+    state::{claim_data::ClaimData, distribution::Distribution},
+};
+
+#[derive(Accounts)]
+pub struct ReclaimUserData<'info> {
+    distribution: Account<'info, Distribution>,
+
+    #[account(
+        mut,
+        close = caller,
+    )]
+    claim_data: Account<'info, ClaimData>,
+
+    /// CHECK: Not read
+    caller: AccountInfo<'info>,
```

Comment:
if anyone can close the ClaimData accounts it sounds like everyone would race to close all of them once the distribution is fully_claimed?

Reply:
I wasn't sure who should be able to receive the funds. The owner? The distribution admin? It's the owner who pays for them I guess.